src/OrchardCore.Modules/OrchardCore.Users/Controllers/AdminController.cs

```diff
@@ -82,7 +82,7 @@ public async Task<ActionResult> Index(UserIndexOptions options, PagerParameters
                 options = new UserIndexOptions();
             }
 
-            var users = _session.Query<User, UserIndex>();
+            var users = _userManager.Users.Select(u => u as User);
```

Comment:
I would not change this file. What you are doing here is resolving the entire list of users then reducing the list with linq for the paging. Makes no sense.